lib/features/composer/presentation/composer_controller.dart

```diff
@@ -216,6 +217,8 @@ class ComposerController extends BaseController {
   @override
   void onClose() {
     _initTextEditor = null;
+    _textEditorWeb = null;
+    viewState.value = Right(UIClosedState());
```

Comment:
Please test additional cases of `reply/forward` with long email, large inline image. If I remember correctly we have some similar variables to store the email content. Please clean it up when composer closes.

Reply:
Will demo on part 2 due to the open-email-memory-leak is not resolved in this PR